libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationTest.java

```diff
@@ -28,199 +28,257 @@
 
 public class MapboxNavigationTest extends BaseTest {
 
-  private MapboxNavigation navigation;
+  @Test
+  public void sanityTest() {
+    MapboxNavigation navigation = buildMapboxNavigation();
 
-  @Before
-  public void before() throws Exception {
-    navigation = new MapboxNavigation(mock(Context.class), ACCESS_TOKEN, mock(NavigationTelemetry.class),
-      mock(LocationEngine.class));
+    assertNotNull(navigation);
   }
 
   @Test
-  public void sanityTest() {
-    assertNotNull("should not be null", navigation);
+  public void sanityTestWithOptions() {
     MapboxNavigationOptions options = MapboxNavigationOptions.builder().build();
-    MapboxNavigation navigationWithOptions = new MapboxNavigation(mock(Context.class),
-      ACCESS_TOKEN, options, mock(NavigationTelemetry.class), mock(LocationEngine.class));
-    assertNotNull("should not be null", navigationWithOptions);
+    MapboxNavigation navigationWithOptions = buildMapboxNavigationWithOptions(options);
+
+    assertNotNull(navigationWithOptions);
   }
 
   @Test
   public void voiceMilestone_onInitializationDoesGetAdded() throws Exception {
-    assertTrue(navigation.getMilestones().get(0).getIdentifier() == VOICE_INSTRUCTION_MILESTONE_ID);
+    MapboxNavigation navigation = buildMapboxNavigation();
+
+    int identifier = navigation.getMilestones().get(0).getIdentifier();
+
+    assertEquals(identifier, VOICE_INSTRUCTION_MILESTONE_ID);
```

Comment:
This should be the other way around, "expected" value 👉 `VOICE_INSTRUCTION_MILESTONE_ID` which would be better represented using the literal value instead of using the constant (👀 [Expect Literals](https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=61)), "actual" value 👉 `identifier`.
The same thing happens in `bannerMilestone_onInitializationDoesGetAdded` test 👇